artifacts/definitions/Windows/NTFS/MFT.yaml

```diff
@@ -51,48 +51,57 @@ parameters:
 
 sources:
   - query: |
-      -- firstly set timebounds for performance
-      LET DateAfterTime <= if(condition=DateAfter,
-            then=DateAfter, else="1600-01-01")
-      LET DateBeforeTime <= if(condition=DateBefore,
-            then=DateBefore, else="2200-01-01")
-
+      -- time testing
+      LET time_test(stamp) = 
```

Comment:
Do we want to extract that into a module?

Reply:
definitely worth adding in I think